wandb/sdk/artifacts/artifact.py

```diff
@@ -463,24 +456,19 @@ def source_collection(self) -> ArtifactCollection:
 
     @property
     def type(self) -> str:
-        """The artifact's type."""
+        """The artifact's type. Common types include `dataset` or `model`."""
         return self._type
 
     @property
     def description(self) -> Optional[str]:
-        """The artifact description.
-
-        Free text that offers a user-set description of the artifact.
+        """A description of the artifact.
         """
         return self._description
 
     @description.setter
     def description(self, description: Optional[str]) -> None:
         """Set the description of the artifact.
```

Comment:
```suggestion
        """Set the description of the artifact. For model or dataset Artifacts, add documentation for your standardized team model or dataset card. In the W&B UI the description will be rendered in markdown. 
```

Reply:
what's the status of this change? I see the comment is not resolved so wanted to check